examples/python/template/README.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Template
+tags: [kebab-case, comma, separated]
+python: https://github.com/rerun-io/rerun/tree/latest/examples/python/template/main.py
+rust: https://github.com/rerun-io/rerun/tree/latest/examples/rust/template/src/main.rs
+---
+
+<!-- Place a screenshot here -->
```

Comment:
Link to how we upload screenshots?

Reply:
We don't have this documented anywhere outside of an issue, so I included some instructions directly in there

Reply:
It's a good point. I'll add this bit of information in `just upload --help` so we can refer to that.

Reply:
You may review/merge https://github.com/rerun-io/rerun/pull/2454 at your convenience so you can update that part.